package.json

```diff
@@ -41,7 +41,8 @@
     "helmet": "^3.21.1",
     "morgan": "^1.9.1",
     "ngx-webstorage-service": "^4.1.0",
-    "rxjs": "~6.5.0",
+    "puppeteer-core": "^1.20.0",
```

Comment:
@tbg-jasonfung  I don't think we need to add this? Other branch was passing without it.

Reply:
@NoelLH Went a bit import trigger happy, have removed now